app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people: list[Person]) -> list[Person]:
+    result_list = []
+    for person in people:
+        result_list.append(Person(person.get("name"), person.get("age")))
+    for person in people:
+        if person.get("wife"):
+            Person.people[person.get("name")].wife = \
+                Person.people[person.get("wife")]
+        if person.get("husband"):
+            Person.people[person.get("name")].husband = \
+                Person.people[person.get("husband")]
```

Comment:
Do not use `\` to split lines. If statement is long and too complex - use variables